plugins/foreman_templates/4.0/index.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: plugin
+title: Foreman Templates 4.0 Manual
+version: 4.0
+---
+
+# 1. {{ page.title }}
+
+This plugin enables synchronization of provisioning templates, partition tables and job templates from external git repository and/or file system. It can be used both with the [community templates](https://github.com/theforeman/community-templates) repository or with any other source of templates that has the same directory structure. Please note that Foreman core provides tracking of changes, browsing the history and reverting natively, but if you need more advanced tools such as branching, code reviews etc. you might be interested in this plugin.
+
+You can use the native git support for simple synchronization with remote git repositories, which is usually useful for syncing new template versions from community repository or for exporting local changes to contribute back. Or if you have more complicated workflow, you can use this plugin to sync the directory when required. This way, you can use any VCS (including git) to control the versioning of that directory and use this plugin to sync whenever the directory is at desired version.
+
+# 2. Installation
+
+You can install the plugin using the foreman-installer (recommended way). To do that, run the installer with following argument
+
+    foreman-installer --enable-foreman-plugin-templates
+
+If you installed the plugin using package directly, you should run the migrations and seed (just to be sure) and restart the Foreman.
```

Comment:
I would prefer not documenting it in details. foreman-installer is the recommended way. We also don't document you could do `git clone ...`